### PR TITLE
Issue and pull request templates improvements

### PR DESCRIPTION
**Pull request by desrosj**
## What?
This adds issue templates for the repository to help bucket new reports into groups.

It also improves on the pull request template.

### CODE REVIEW

File: .github/ISSUE_TEMPLATES/1-bug-report.yml
Comment (jeffpaul) [inline .github/ISSUE_TEMPLATES/1-bug-report.yml]: this doesn't match the current bug label, so either update here or update the repo label

File: .github/ISSUE_TEMPLATES/1-bug-report.yml
Comment (jeffpaul) [inline .github/ISSUE_TEMPLATES/1-bug-report.yml]: might be able to collapse this into the question above as they're directly related and reduces a required field by 1 for a reporter

File: .github/ISSUE_TEMPLATES/1-bug-report.yml
Comment (jeffpaul) [inline .github/ISSUE_TEMPLATES/1-bug-report.yml]: Might be helpful to use a permalink to show where someone can find the version in case they're not as versed on that specific and are reporting a bug from personal usage, otherwise the data we get in this required field might not be helpful/accurate

File: .github/ISSUE_TEMPLATES/1-bug-report.yml
Comment (jeffpaul) [inline .github/ISSUE_TEMPLATES/1-bug-report.yml]: again, a permalink example in the description could be helpful or alternatively updating the placeholder to be said permalink

File: .github/ISSUE_TEMPLATES/1-bug-report.yml
Comment (jeffpaul) [inline .github/ISSUE_TEMPLATES/1-bug-report.yml]: again, a permalink example in the description could be helpful or alternatively updating the placeholder to be said permalink

File: .github/ISSUE_TEMPLATES/2-feature-request.yml
Comment (jeffpaul) [inline .github/ISSUE_TEMPLATES/2-feature-request.yml:3]: this doesn't match the current `enhancement` label, so either update here or update the repo label

File: .github/ISSUE_TEMPLATES/3-request-to-help-implement.yml
Comment (jeffpaul) [inline .github/ISSUE_TEMPLATES/3-request-to-help-implement.yml:3]: this doesn't match the current `help wanted` label, so either update here or update the repo label

File: .github/ISSUE_TEMPLATES/2-feature-request.yml
Comment (jeffpaul) [inline .github/ISSUE_TEMPLATES/2-feature-request.yml]: `This action has a specific purpose, make tracking all contributions the WordPress project on Github easier to track collect in an automated way.` >> `This action has a specific purpose, to make tracking all contributions to the WordPress project on GitHub easier to track and collect in an automated way.`

File: .github/ISSUE_TEMPLATES/3-request-to-help-implement.yml
Comment (jeffpaul) [inline .github/ISSUE_TEMPLATES/3-request-to-help-implement.yml:23]: "WordPress-flavor" is not a clear phrase, perhaps we can borrow some text from the handbook page on the topic?

**Review by jeffpaul — CHANGES_REQUESTED** [review]
Some questions, but at least need to handle the label issues in the 3 issue templates before merging  this in

File: .github/ISSUE_TEMPLATES/1-bug-report.yml
Comment (desrosj) [reply .github/ISSUE_TEMPLATES/1-bug-report.yml]: Gone and cleaned up our labels here.

File: .github/ISSUE_TEMPLATES/2-feature-request.yml
Comment (desrosj) [reply .github/ISSUE_TEMPLATES/2-feature-request.yml]: Much better. I did remove the second "track" in the sentence though. It felt repetitive.

File: .github/ISSUE_TEMPLATES/3-request-to-help-implement.yml
Comment (desrosj) [reply .github/ISSUE_TEMPLATES/3-request-to-help-implement.yml:23]: This is the only one I'm unsure of. Do you have any suggestions?

Comment (jeffpaul) [reply .github/ISSUE_TEMPLATES/3-request-to-help-implement.yml:23]: I'd say fine enough for now, perhaps someone else will come along with better copy later but no reason to hold this up

**Review by jeffpaul — APPROVED** [review]

**Pull request merged**
